client/app/telecom/pack/xdsl/tasks/packs-xdsl-tasks.js

```diff
@@ -0,0 +1,13 @@
+angular.module('managerApp').config(($stateProvider) => {
+  $stateProvider.state('telecom.pack.xdsl.tasks', {
+    url: '/tasks',
+    views: {
+      'xdslView@telecom.pack.xdsl': {
+        templateUrl: 'app/telecom/pack/xdsl/tasks/pack-xdsl-tasks.html',
+        controller: 'XdslTasksCtrl',
+        controllerAs: 'XdslTasks',
```

Comment:
Why not prefer `$ctrl`? We tend to homogenize that as much as we can:)

Reply:
It's also the default name for controllers in components and we should tend to using components